### PR TITLE
Add benchmark code - Closes #694

### LINKED ISSUE #694: Add socket.io benchmark
### Description

Add socket.io benchmark that acts as a part of integration test suite.

### Acceptance Criteria

- Benchmark the following routes: blocks, transactions, accounts.

### PR DESCRIPTION
**Pull request by MichalTuleja**
### What was the problem?
This PR resolves #694

### How was it solved?

- [x] Added benchmark for transactions
- [x] Added benchmark for blocks
- [x] Added benchmark for accounts
- [x] Updated known account list

### How was it tested?
Locally

### CODE REVIEW

File: tests/package.json [hunk 1]
```diff
@@ -30,9 +30,11 @@
     "test:integration:APIv1:SDKv4": "jest --config=jest.config.integration.apiv1.sdkv4.js",
     "test:integration:APIv2:SDKv5": "jest --config=jest.config.integration.apiv2.sdkv5.js",
     "test:functional": "jest --config=jest.config.functional.js",
+    "benchmark": "artillery run benchmark/socket_rpc_v2.yaml",
     "start": "node app.js"
   },
   "dependencies": {
+    "artillery": "^1.7.7",
```
Comment (sameersubudhi) [inline tests/package.json:37]: Should we not install `artillery` under `devDependencies`?

File: tests/benchmark/socket_rpc_v2.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+          data: {"method": "get.blocks", "params": {"limit": 10}}
+      - emit:
+          channel: "request"
+          data: {"method": "get.blocks", "params": {"limit": 100}}
+  - name: "Delegates"
+    engine: "socketio"
+    flow:
+      - emit:
+          channel: "request"
+          data: {"method": "get.delegates", "params": {"limit": 101}}
```
Comment (nagdahimanshu) [inline tests/benchmark/socket_rpc_v2.yaml:36]: For v2, we have `get.accounts` with params `isDelegate` for delegates

**Review by nagdahimanshu — APPROVED** [review]
Looks good 👍

**Review by sameersubudhi — APPROVED** [review]
LGTM 👍

**Pull request merged**